cime_config/buildnml

```diff
@@ -567,8 +567,10 @@ def buildnml(case, caseroot, component):
     cimeroot = case.get_value("CIMEROOT")
     fd_dir = os.path.join(cimeroot, "src","drivers","nuopc","mediator")
     coupling_mode = case.get_value('COUPLING_MODE')
-    if coupling_mode == 'cesm' or coupling_mode == 'hafs':
+    if coupling_mode == 'cesm':
         filename = os.path.join(fd_dir,"fd_cesm.yaml")
+    elif coupling_mode == 'hafs':
+        filename = os.path.join(fd_dir,"fd_hafs.yaml")
     else:
```

Comment:
this should probably test for coupling_mod == 'nems' and add a final block to just error out if coupling_mode is something else.

Reply:
It is fixed. I think it was still safe because COUPLING_MODE XML variable only expects cesm, nems and hafs.